pandas/core/groupby/groupby.py

```diff
@@ -906,22 +906,18 @@ def pipe(
         return com.pipe(self, func, *args, **kwargs)
 
     @final
-    def get_group(self, name, obj=None) -> DataFrame | Series:
+    def get_group(self, name) -> DataFrame | Series:
         """
         Construct DataFrame from group with provided name.
 
         Parameters
         ----------
         name : object
             The name of the group to get as a DataFrame.
-        obj : DataFrame, default None
-            The DataFrame to take the DataFrame out of.  If
-            it is None, the object groupby was called on will
-            be used.
 
         Returns
         -------
-        same type as obj
+        DataFrame
```

Comment:
Small issue: I think that it returns Series or DataFrame

Reply:
thanks, I rolled back my changes there completely.